src/server.c

```diff
@@ -1481,7 +1481,7 @@ void qd_server_run(qd_dispatch_t *qd)
     int i;
     assert(qd_server);
     assert(qd_server->container); // Server can't run without a container
-    qd_log(LOG_SERVER, QD_LOG_NOTICE, "Operational, %d Threads Running (process ID %ld)",
+    qd_log(LOG_SERVER, QD_LOG_INFO, "Operational, %d Threads Running (process ID %ld)",
```

Comment:
Not a critique of this patch, but I've found this log message (and the "shutdown" message on line 1515) useful in many cases. In fact the "shutdown" log is very useful to determine if the router had a controlled shutdown ("shutdown" will be in the log) vs. a kill -9 or a truncated logfile ("shutdown" not in the log).

This makes me think that It Would Be Nice (TM) to have status+configuration log messages that only get emitted once at startup and shutdown that should be enabled by default.  But that's probably better left to a separate effort from this patch.

Not a big deal - unless others feel it should stay.

Reply:
> useful in many cases. In fact the "shutdown" log is very useful to determine if the router had a controlled

I will leave the router "Shut Down" log message at INFO level so it is visible by default

Reply:
> But that's probably better left to a separate effort from this patch.

Yes, I agree that the status+configuration one time log messages should be INFO level. I will put out another PR which will try to do this. Thx